aea/protocols/generator/base.py

```diff
@@ -991,16 +991,28 @@ def _dialogue_class_str(self) -> str:
         cls_str += (
             self.indent + "from typing import Callable, FrozenSet, Type, cast\n\n"
         )
-        cls_str += (
-            self.indent
-            + "from aea.protocols.dialogue.base import Dialogue, DialogueLabel, Dialogues\n"
-        )
         cls_str += self.indent + "from aea.mail.base import Address\n"
-        cls_str += self.indent + "from aea.protocols.base import Message\n\n"
-        cls_str += self.indent + "from {}.message import {}Message\n".format(
-            self.dotted_path_to_protocol_package,
-            self.protocol_specification_in_camel_case,
-        )
+        cls_str += self.indent + "from aea.protocols.base import Message\n"
+        if "aea.protocols.dialogue" < self.dotted_path_to_protocol_package:
+            cls_str += (
+                    self.indent
+                    + "from aea.protocols.dialogue.base import Dialogue, DialogueLabel, Dialogues\n"
+            )
+            if not self.dotted_path_to_protocol_package.startswith("aea"):
```

Comment:
I think the conditionals are not needed; the protocol generator only generates `package...` protocols. For the aea protocols we manually update the import paths!

Reply:
With this, we can later add a test that checks all protocol packages, including those under `aea`, if they are the result of the latest generator and detect if there is any manual tampering with the protocol package code. 

This would be similar to what we do with `t_protoocol`, but for all protocols.

Reply:
We don't add incomplete functionality; it is just confusing and adds to the bloat. We can add this when we need it. Please revert.